src/applications/facility-locator/containers/FacilityDetail.jsx

```diff
@@ -25,6 +25,22 @@ class FacilityDetail extends Component {
     focusElement('.va-nav-breadcrumbs');
   }
 
+  componentDidUpdate(prevProps) {
+    const justLoaded =
+      prevProps.currentQuery.inProgress && !this.props.currentQuery.inProgress;
```

Comment:
Oof, took me a while to realize `currentQuery.inProgress` is used for both the search results and the facility data lookup...

Reply:
Yeah...The Facility Locator could use some cleanup. I only knew because of the `mapStateToProps` below.